source/_includes/custom/search.html

```diff
@@ -18,5 +18,11 @@
       entries : '.entries',         // selector for search entries containing element (contained within results above)
       template: '#search-results-template'  // selector for Mustache.js template
     });
+    $('#search-query').keydown(function(event){
```

Comment:
Sure! Sorry, I forgot I had this open 😅 
https://github.com/cerner/cerner.github.io/pull/66/commits/38febbd2f1031baa2b0cab3cf213d81fde154440